pysatNASA/instruments/omni_hro.py

```diff
@@ -140,59 +139,12 @@ def clean(self):
                                file_cadence=pds.DateOffset(months=1))
 
 # Set the list_remote_files routine
-remote_dir = '/pub/data/omni/omni_cdaweb/hro_{tag:s}/{{year:4d}}/'
-download_tags = {inst_id: {tag: {'remote_dir': remote_dir.format(tag=tag),
-                                 'fname': supported_tags[inst_id][tag]}
-                           for tag in tags.keys()}
-                 for inst_id in inst_ids.keys()}
-list_remote_files = functools.partial(cdw.list_remote_files,
-                                      supported_tags=download_tags)
-
-
-# Set the download routine
-def download(date_array, tag, inst_id, data_path, update_files=False):
-    """Download OMNI HRO data from CDAWeb.
-
-    Parameters
-    ----------
-    date_array : array-like
-        Sequence of dates for which files will be downloaded.
-    tag : str
-        Denotes type of file to load.
-    inst_id : str
-        Specifies the satellite ID for a constellation.
-    data_path : str
-        Path to data directory.
-    update_files : bool
-        Re-download data for files that already exist if True (default=False)
-
-    Raises
-    ------
-    IOError
-        If a problem is encountered connecting to the gateway or retrieving
-        data from the repository.
-
-    Warnings
-    --------
-    Only able to download current forecast data, not archived forecasts.
-
-    Note
-    ----
-    Called by pysat. Not intended for direct use by user.
-
-    """
-
-    # Set the download tags
-
-    # Adjust the date_array for monthly downloads
-    if date_array.freq != 'MS':
-        date_array = pysat_time.create_date_range(
-            dt.datetime(date_array[0].year, date_array[0].month, 1),
-            date_array[-1], freq='MS')
+download_tags = {'': {'1min': 'OMNI_HRO_1MIN', '5min': 'OMNI_HRO_5MIN'}}
+download = functools.partial(cdw.cdas_download,
+                             supported_tags=download_tags)
```

Comment:
Is the monthly cadence maintained?

Reply:
It should be handled by cdasws. I'll put together an example to demonstrate this.

Reply:
So here's an example of me feeding a bunch of different dates to the remote_file_list method.
Giving it any day within a month  will get you that month's file.
Giving it a range with multiple months, will get you all of the monthly files.
You can also give it a range of dates within a month, and it will return the file for that month.
I think this is good behavior.

I'm glad you asked about this, because I did find a bug while testing this. 
After I pull in the suggested changes above, I'll push up my fix with the test.

```
In [1]: import pysat

In [2]: hro = pysat.Instrument(platform='omni', name='hro', tag='1min')

In [3]: import datetime as dt

In [4]: d1 = dt.datetime(1995,1,1)

In [5]: hro.remote_file_list(start=d1)
Out[5]:
1995-01-01    omni_hro_1min_19950101_v01.cdf
dtype: object

In [6]: d1 = dt.datetime(1995,1,15)

In [7]: hro.remote_file_list(start=d1)
Out[7]:
1995-01-01    omni_hro_1min_19950101_v01.cdf
dtype: object

In [8]: d1 = dt.datetime(1995,1,25)

In [9]: hro.remote_file_list(start=d1)
Out[9]:
1995-01-01    omni_hro_1min_19950101_v01.cdf
dtype: object

In [10]: d1 = dt.datetime(1995,2,1)

In [11]: hro.remote_file_list(start=d1)
Out[11]:
1995-02-01    omni_hro_1min_19950201_v01.cdf
dtype: object

In [12]: d2 = dt.datetime(1995,3,2)

In [13]: hro.remote_file_list(start=d1, stop=d2)
Out[13]:
1995-02-01    omni_hro_1min_19950201_v01.cdf
1995-03-01    omni_hro_1min_19950301_v01.cdf
dtype: object
```